orchagent/dash/dashaclgroupmgr.cpp

```diff
@@ -492,12 +492,6 @@ task_process_status DashAclGroupMgr::updateRule(const string& group_id, const st
 {
     SWSS_LOG_ENTER();
 
-    if (isBound(group_id))
```

Comment:
How do you ensure this from requirements?

_It is not permitted to modify rules within a group that is currently bound to an ENI._